core/templates/dev/head/pages/exploration_editor/editor_navigation_directive.html

```diff
@@ -81,3 +81,28 @@
     {% endif %}
   </ul>
 </script>
+<style type="text/css">
+.popover{
+  font-size: 13px;
+  max-width: 300px;
+  text-align: center;
+  z-index: -1;
+  color: white;
+  background-color: black;
+}
+.popover.bottom > .arrow{
+  top: -11px;
+  left: 50%;
+  margin-left: -11px;
```

Comment:
@shivanXI 
Please note that I am not working on visibility but just the cut-off fix for the .popover class.
As of now this PR commits duplicate all the properties already present in third party. 

![selection_002](https://cloud.githubusercontent.com/assets/24438869/24782526/704f9cb8-1b64-11e7-88fd-ed375dead91d.png)


Regarding point 2, 

> But the changes done above in no. 1 reason are not enough for changing .... 

Have you tested it, before commenting If not please do? CSS properties don't work that way.

Edit: I have checked out your branch and modified the code to remove the duplicates. You can check the chrome dev tools how override works. 

![screenshot from 2017-04-07 07-23-53](https://cloud.githubusercontent.com/assets/24438869/24782419/b6313198-1b63-11e7-9c68-160ed338ee2d.png)
As you can see here the code of color and background are **only** modified preventing duplicate code. 

Here is the[ Gist file](https://gist.github.com/shaz13/04c72e591a4c1ceccd33abdea8b264b5) of working code without duplicate CSS properties